<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="log_table">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>511</width>
     <height>311</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>운동 날짜</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>운동 분야</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>시간</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>트레이너 인증</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="info_label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>*트레이너 인증을 완료해야 운동 현황에 반영*</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="date_edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>125</width>
     <height>24</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2020</year>
     <month>7</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="date">
    <date>
     <year>2020</year>
     <month>7</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>운동 기록 조회 및 트레이너 인증</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
